src/components/vegetation-graph.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useState } from "react";
+import { observer } from "mobx-react";
+import { useStores } from "../use-stores";
+import { Chart as ChartJS, CategoryScale, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Filler, Legend, defaults } from "chart.js";
+import { Line, Bar } from "react-chartjs-2";
+import { Vegetation } from "../types";
+
+import cssExports from "./common.scss";
+
+defaults.font.family = cssExports.robotoFontFamily;
```

Comment:
So does this also set the defaults for all other files that import from chart.js?